src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -194,9 +195,9 @@
   ([label props-value]
    (details-list-row label props-value nil))
   ([label props-value extra-props-value]
-   (let [[props value]             (if (string? props-value)
-                                     [nil props-value]
-                                     props-value)
+   (let [[props value] (if (string? props-value)
```

Comment:
we don't align anymore ?

Reply:
i think we never align in `let`?

Reply:
when did we stop ? afaik we've always done it (not that I like it tbh).